Library/Homebrew/hardware.rb

```diff
@@ -55,6 +55,8 @@ def self.cores_as_words
     when 1 then "single"
     when 2 then "dual"
     when 4 then "quad"
+    when 6 then "hexa"
```

Comment:
I'm not sure that's true, but maybe some non-Europeans like @ilovezfs can comment on that. I've been doing some Googling and checked relevant Wikipedia articles in English and those prefixes were all over the place.